### PR TITLE
Enhance RssAgent with the `events_order` option

### PR DESCRIPTION
**Pull request by knu**
#918 did not actually address #907 which was about RssAgent, so here is the fix for the issue.

### CODE REVIEW

**Comment by knu** [comment]
I didn't use `can_order_created_events!` which didn't apply cleanly to RssAgent, which has its own tricks for event deduplication and capping. (`seen_ids` and `max_events_per_run`)

File: app/models/agents/rss_agent.rb [hunk 1]
```diff
@@ -77,26 +90,16 @@ def check
...
-                last_updated: entry.last_updated,
-                url: entry.url,
-                urls: entry.urls,
-                description: entry.description,
-                content: entry.content,
-                title: entry.title,
-                authors: entry.authors,
-                categories: entry.categories
-              })
+              Rails.logger.info event.inspect
```
Comment (cantino) [inline app/models/agents/rss_agent.rb]: Do you want to leave this logger?

File: app/models/agents/rss_agent.rb [hunk 2]
```diff
@@ -122,5 +125,22 @@ def check_and_track(entry_id)
         true
       end
     end
+
+    def feed_to_events(feed)
+      feed.entries.map { |entry|
+        Event.new(payload: {
```
Comment (cantino) [inline app/models/agents/rss_agent.rb]: Does `create_event` accept an event object and assign the Agent and User in master right now?

**Comment by cantino** [comment]
Looks good!

File: app/models/agents/rss_agent.rb (hunk 1 above)
Comment (knu) [inline app/models/agents/rss_agent.rb]: Oops, will delete the line.

File: app/models/agents/rss_agent.rb (hunk 2 above)
Comment (knu) [inline app/models/agents/rss_agent.rb]: Yes, and this PR is based on the required commit.

Comment (cantino) [inline app/models/agents/rss_agent.rb]: Okay :)

**Pull request merged**